webapp/src/client/client.js

```diff
@@ -11,22 +11,16 @@ export default class Client {
         this.url = url + '/plugins/' + id;
     }
 
-    startMeeting = async (channelId, rootId, personal = true, topic = '', meetingId = 0, force = false) => {
+    startMeeting = async (channelId, rootId, personal = true, topic = '', force = false) => {
```

Comment:
Seems like `personal` is always `true` as well right? I think these were added as placeholders, but they haven't been implemented for years so I think they are a bit of tech debt at this point. Do you see value in keeping it around? I don't want to add risk to the PR so I trust your judgment

Reply:
@mickmister I don't think we need this for now. I have removed it. We can always add it back if we need it in the future.